# HG changeset patch
# User Rik <rik@octave.org>
# Date 1438204706 25200
#      Wed Jul 29 14:18:26 2015 -0700
# Node ID 517f5a006279715cede6043aea73fc032329fd2e
# Parent  4300fe896004de0391c94c142515953f1113d091
build: Don't delete ChangeLog with distclean target (bug #45646).

* Makefile.am: Move ChangeLog to MAINTAINERCLEANFILES variable.

diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -156,20 +156,20 @@ noinst_SCRIPTS = run-octave
 
 OCTAVE_INTERPRETER_TARGETS += run-octave
 
 CLEANFILES += \
   $(BUILT_SOURCES)
 
 DISTCLEANFILES += \
   .gdbinit \
-  ChangeLog \
   $(DIRSTAMP_FILES)
 
 MAINTAINERCLEANFILES += \
+  ChangeLog \
   $(BUILT_DISTFILES)
 
 CONFIG_FILES = @ac_config_headers@ @ac_config_files@
 
 octinclude_HEADERS += oct-conf-post.h
 nodist_octinclude_HEADERS += config.h
 
 all-local: $(ALL_LOCAL_TARGETS) $(noinst_SCRIPTS) $(DIST_INFO_FILES) .gdbinit $(DOC_TARGETS)
